pkgs/development/python-modules/orbax-checkpoint/default.nix

```diff
@@ -91,7 +91,7 @@ buildPythonPackage rec {
   meta = {
     description = "Orbax provides common utility libraries for JAX users";
     homepage = "https://github.com/google/orbax/tree/main/checkpoint";
-    changelog = "https://github.com/google/orbax/releases/tag/v${version}";
+    changelog = "https://github.com/google/orbax/blob/${lib.removePrefix "refs/tags" src.rev}/checkpoint/CHANGELOG.md";
```

Comment:
```suggestion
    changelog = "https://github.com/google/orbax/blob/${lib.removePrefix "refs/tags/" src.rev}/checkpoint/CHANGELOG.md";
```

Reply:
This ended up being harder than I thought X)